src/__smoke-test__/dataplane-integ.spec.ts.ts

```diff
@@ -0,0 +1,83 @@
+import { test, expect } from '@playwright/test';
+import {
+    PERFORMANCE_NAVIGATION_EVENT_TYPE,
+    PERFORMANCE_RESOURCE_EVENT_TYPE,
+    PAGE_VIEW_EVENT_TYPE,
+    SESSION_START_EVENT_TYPE
+} from '../plugins/utils/constant';
+
+// Environment variables set through CLI command
```

Comment:
Did some research to find if Playwright accepts custom parameters from the command line, and the short answer is Playwright does not support this option.

Several feature requests were raised to request this, but the Playwright team stated that using env variables is the preferred option at the moment.

As a result, the proposed solution is the following:

define the npm command as "cross-env URL=$URL MONITOR_ID=$MONITOR ENDPOINT=$ENDPOINT npx playwright test"
when we run npm run smoke, we pass in the parameters as URL=test_url.com MONITOR=monitor_name ENDPOINT=endpoint npm run smoke